src/Rollbar.php

```diff
@@ -87,12 +87,25 @@ public static function scope($config)
         return self::$logger->scope($config);
     }
 
-    public static function log($level, $toLog, $extra = array(), $isUncaught = false)
+    public static function log($level, $toLog, $extra = array())
     {
         if (is_null(self::$logger)) {
             return self::getNotInitializedResponse();
         }
-        return self::$logger->log($level, $toLog, (array)$extra, $isUncaught);
+        return self::$logger->log($level, $toLog, (array)$extra);
+    }
+
+    /**
+     * @since 3.0.0
+     */
+    public static function logUncaught($level, Throwable $toLog, $extra = array())
```

Comment:
I'm on the fence about this method. It makes for an easy upgrade path: code doing `Rollbar::log($level, $data, $context, true)` just gets rewritten to `Rollbar::logUncaught($level, $data, $context)`, provided that the `$data` is an instance of `Throwable` (which it must be, since the only "uncaught" things derive logically from `Throwable`).

However, if no-one is using `Rollbar::log` with the fourth parameter, then this is just code bloat. Without metrics, I'm erring on the side of caution and providing an upgrade path for the breaking change.

Reply:
There was at least one person who intended to use it 3 years ago. https://github.com/rollbar/rollbar-php/pull/314

In general the SDKs allow specifying "uncaught", and it's a common pattern to use this flag in a check ignore handler, if not elsewhere. It makes sense to continue to provide a way to set it.

I'd agree this should only apply to Throwable. Hopefully no one had been using it differently.